src/style.css

```diff
@@ -1 +1,26 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
According to the checklist, you should not use tag names for styling (except `html` and `body`). It's better to add a specific class to the body and style it.